### ISSUE TITLE
Possible bug in template: Going back reveals lorem ipsum license

### ISSUE THREAD

**Issue by nbigaouette-eai** [volks73/cargo-wix#77]
I've followed the directions in the README to create an msi. I have barely tweaked the XML automatically generated (change project name, binary to install, etc.)

I haven't adapted the XML to provide a license.

When I'm in the _Custom Setup_ dialog where I can select the installation path, clicking _Back_ reveals an EULA dialog with a lorem ipsum text. I cannot click next until I accept that license or I can simply click back to come back to the initial dialog. From there _Next_ brings me to the _Custom Setup_ again, without going through the EULA.

I guess it's the template that still contains a reference to the EULA dialog?

**Comment by volks73** [comment]
It sounds like the mechanism for skipping the License Agreement dialog for the [WixUI_FeatureTree](https://wixtoolset.org//documentation/manual/v3/wixui/dialog_reference/wixui_featuretree.html) configuration is not working correctly for the `Back` button from the Customize dialog. The natural progression is: 

```text
Welcome -> License Agreement -> Customize -> Verify Ready -> Progress -> ...
```

and the non-EULA progression is: 

```text
Welcome -> Customize -> Verify Ready -> Progress -> ...
```

So, the `Next` button from the Welcome dialog should jump to the Customize dialog and the `Back` button for the Customize dialog should go back to the Welcome dialog, as outlined for the [WixUI_InstallDir](https://wixtoolset.org//documentation/manual/v3/wixui/dialog_reference/wixui_installdir.html) configuration. The License dialog is not actually removed, but instead, it is just skipped over.

I was able to reproduce the same behavior you are describing. It appears related to the `Order` attribute of the [`Publish`](https://wixtoolset.org/documentation/manual/v3/xsd/wix/publish.html) element. See the comment by user922020 in the [Removing License Dialog](https://stackoverflow.com/questions/13045658/removing-license-dialog) StackOverflow Q&A:

>I recommend a higher order that "2". "2" works here because the OP went one-higher than the published source code. But who wants to read the source? Other dialogs of other WixUI can use an order that maxes out at 4. Because Order means something like weight - higher has precedence. I would use Order="99" for anything behavior you wish to override. The only higher order in the published source is the Finish button on ExitDialog which has Order 999.

In your WXS file, there should be the following two lines (around Line 172 depending on other modifications and options enabled/disabled within the WXS file) that implement the `Next` and `Back` buttons, respectively:

```xml
<Publish Dialog='WelcomeDlg' Control='Next' Event='NewDialog' Value='CustomizeDlg' Order='2'>1</Publish>
<Publish Dialog='CustomizeDlg' Control='Back' Event='NewDialog' Value='WelcomeDlg' Order='2'>1</Publish>
```

Please change the `Order` attribute value to `99` like so:

```xml
<Publish Dialog='WelcomeDlg' Control='Next' Event='NewDialog' Value='CustomizeDlg' Order='99'>1</Publish>
<Publish Dialog='CustomizeDlg' Control='Back' Event='NewDialog' Value='WelcomeDlg' Order='99'>1</Publish>
```

This should resolve the issue for your specific WXS file and MSI. I am in the process of updating the embedded template and pushing a new update.

Thank you for the bug report!

**Comment by volks73** [comment]
Resolved as of ae5ee24252a6cb643b8059242503d17abe2a8976
